docker/2.0/Dockerfile

```diff
@@ -34,7 +34,7 @@ VOLUME /var/lib/mysql
 WORKDIR /mattermost
 
 # Copy over files
-ADD https://github.com/mattermost/platform/releases/download/v2.0.0-rc2/mattermost.tar.gz /
+ADD https://github.com/mattermost/platform/releases/download/v2.0.0/mattermost.tar.gz /
```

Comment:
Hey @crspeller I don't remember making this change, is this okay? I think this got added from another commit.